### PR TITLE
:recycle: rename `begin` and `end` to better distinguish

### PR DESCRIPTION
**Pull request by MrKrisKrisu**

### CODE REVIEW

File: app/Notifications/EventSuggestionProcessed.php [hunk 1]
```diff
@@ -32,9 +32,9 @@ public function via(): array {
 
     public function toArray(): array {
         return [
-            'accepted'      => $this->event !== null,
-            'event'         => $this->event?->only(['id', 'slug', 'name', 'begin', 'end']),
-            'suggestedName' => $this->eventSuggestion->name,
+            'accepted'        => $this->event !== null,
+            'event'           => $this->event?->only(['id', 'slug', 'name', 'checkin_start', 'checkin_end']),
```
Comment (marhei) [inline app/Notifications/EventSuggestionProcessed.php:36]: Breaking API change without information 👀

Comment (marhei) [reply app/Notifications/EventSuggestionProcessed.php:36]: I mean when you already changed the keys on this API endpoint you should do it everywhere.

Comment (MrKrisKrisu) [reply app/Notifications/EventSuggestionProcessed.php:36]: Oh, well. This wasn't documented (and not tested), so I didn't saw that this is also provided to API.

Comment (marhei) [reply app/Notifications/EventSuggestionProcessed.php:36]: Yes, the returned notification API data is not documented at all. It only says: Just try it!

**Pull request merged**